### DISCUSSION TITLE
Icon/Image Links

### DISCUSSION THREAD

**Discussion by cerulean-tiger** [clientIO/joint#2161] [Q&A]
### Introduction

Is there a demo or example of using a Font Awesome (https://fontawesome.com/icons/circle-info?f=classic&s=solid) icon as a link label or any type of image as a link label?  My understanding is that the label should be replaced with custom markup.  I have tried variations of the following, but nothing is appearing on the link.
`link.appendLabel({
  markup: [{
    tagName: 'i',
    className: "fa-solid fa-circle-info"
  }]
});`

I have also tried alternatively using an image such as:
`link.appendLabel({
  markup: [{
    tagName: "img",
    attributes: {
      src: "/images/my-image.svg"
    }
  }]
});`

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment]
The link label can be an image: https://jsfiddle.net/kumilingus/rdo2g6z9/.

For the _Font Awesome_ icons, you need to use the `<text/>` element. Please see https://jsfiddle.net/kumilingus/b3dnft9z/.

**Reply by kumilingus** [reply]
Also note, _JointJS_ uses primarily SVG. `<i/>` and `<img/>` are HTML elements. The image element in SVG is [`<image/>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image).
